README.md

```diff
@@ -64,8 +66,9 @@ You will need the values from the OIDC client that you created in the previous s
 In your application's [vue-router][] configuration, import the `@okta/okta-vue` plugin and pass it your OpenID Connect client information:
```

Comment:
I have a question about this documentation.
I think we are in the main.ts and not in the [vue-router][] configuration, no?